projects/storefrontapp-e2e-cypress/cypress/integration/regression/variants/apparel-checkout-as-guest.e2e-spec.ts

```diff
@@ -86,7 +86,9 @@ context('Apparel - checkout as guest', () => {
         .should('eq', 200);
 
       cy.get('cx-login div.cx-login-greet').should('exist');
-      cy.get('.cx-checkout-title').should('contain', 'Delivery Address');
+      cy.get('.cx-checkout-title', {
+        timeout: Cypress.config('defaultCommandTimeout') + 20000,
```

Comment:
I don't think we need this anymore as it was timing out due to the fact that we had that infinite loop error in delivery mode, which was fixed [here](https://github.com/SAP/spartacus/commit/c1a04a3d2286b031f72aac6931ee71bfb51d0149)

Reply:
Moreover, the default timeout for the CI e2e tests is 30s, which is already long enough to wait for a response to at least come back from the backend